armada_jupyter/submissions.py

```diff
@@ -119,3 +121,24 @@ def convert_to_submission(file: str) -> Submission:
     return Submission(
         data[YMLSTR.QUEUE], data[YMLSTR.JOB_SET_ID], data[YMLSTR.TIMEOUT], jobs
     )
+
+
+def match_ingress_protobuf_keys(ingress_config: Dict[str, str]) -> Dict[str, str]:
```

Comment:
I would rename this to something like `remap_ingress_protobuf_keys`. `match` doesn't really imply mutation, and given the function signature you might expect this function to leave the original configuration alone and return a new, corrected dict. I'd also drop `return`ing `ingress_config` because it will be mutated and you don't need to re-assign it.